### PR TITLE
Transaction tab constrains updated.

### PR DESCRIPTION
**Pull request by aparthibanpaypal**
Ready for Review

### CODE REVIEW

**Review by pdamodarannair — APPROVED** [review]

**Review by nanbehera — APPROVED** [review]
approved

**Review by jsaini-hw — COMMENTED** [review]
@aparthibanpaypal I know you didn't add it but can you remove unused variable 
`private var uiSegmentedControl: UISegmentedControl?` from the ListReceiptController?
Thank You!

**Review by jsaini-hw — APPROVED** [review]

**Review by pdamodarannair — APPROVED** [review]

**Review by nanbehera — DISMISSED** [review]

**Review by jsaini-hw — DISMISSED** [review]

**Review by pdamodarannair — DISMISSED** [review]

File: Common/Sources/Extensions/UIView.swift [hunk 1]
```diff
@@ -172,4 +172,19 @@ public extension UIView {
...
+            if subview is UILabel {
+                let label: UILabel? = (subview as? UILabel)
+                label?.adjustsFontSizeToFitWidth = true
+                label?.minimumScaleFactor = 0.1
+            } else {
+                adjustWidthOfSegmetTitles(view: subview)
+            }
+        }
+    }
 }
```
Comment (malaw-hw) [inline Common/Sources/Extensions/UIView.swift:190]: is it a typo? SegmetTitles or SegmentTitles?

Comment (aparthibanpaypal) [reply Common/Sources/Extensions/UIView.swift:190]: Yes it is SegmentTitles. Updating it.

**Review by jsaini-hw — APPROVED** [review]

**Review by pdamodarannair — APPROVED** [review]

**Review by nanbehera — APPROVED** [review]

**Review by jsaini-hw — APPROVED** [review]

**Pull request merged**